app/src/main/java/org/astraea/partitioner/nodeLoadMetric/NodeLoadClient.java

```diff
@@ -53,23 +52,19 @@ public NodeLoadClient(Map<String, Integer> jmxAddresses) throws IOException {
    */
   public synchronized Map<Integer, Integer> loadSituation(Cluster cluster)
       throws UnknownHostException {
-    if (overOneSecond() && notInMethod) {
+    if (overOneSecond(lastTime) && notInMethod) {
       notInMethod = false;
       nodesOverLoad(cluster);
+      notInMethod = true;
     }
-    try {
-      countDownLatch.await();
-    } catch (InterruptedException e) {
-      e.printStackTrace();
-    }
+    Utils.waitFor(() -> currentLoadNode != null);
```

Comment:
This method is synchronized. If currentLoadNode is null here, it can't be updated by other synchronized methods

Reply:
This method should not be synchronized.I forget remove it. 
 `Utils.waitFor(() -> currentLoadNode != null);`Its purpose is the method will not return null when it is called multiple times initially

Reply:
Please add comment here.